### DISCUSSION TITLE
Let's get started with Material!

### DISCUSSION THREAD

**Discussion by asyncliz** [material-components/material-web#4172] [Announcements]
## 👋 Welcome to Material Web!

Hi all! We're enabling [GitHub discussions](https://docs.github.com/en/discussions) for this project. If you have a problem or question, try posting here for community feedback before filing a bug or feature request issue.

We can't wait to see what you build! ✨

**Comment by manojadams** [comment]
Hi, does this project accept open source contributions?

**Reply by asyncliz** [reply]
Yes we do! There are a couple things to keep in mind right now though.

1. We don't have a [contribution guide](https://github.com/material-components/material-web/issues/246) yet, so prepare for lots of unknowns.
2. We're short staffed, and reviewing 3P contributions takes time. It may be a slow review process for larger changes.

We recommend small and focused bug fix PRs! Those are the most helpful thing for us right now :)

**Comment by andrew-vdb** [comment]
Is there any web template i can use as starting point?

**Reply by andrew-vdb** [reply]
this project is dead, there was polymer, now this
